java-vtl-script/src/main/java/no/ssb/vtl/script/functions/VTLRound.java

```diff
@@ -53,7 +53,7 @@ public VTLRound() {
             throw new IllegalArgumentException("Number of decimals must be equal to or greater than zero");
         }
 
-        BigDecimal bigDecimal = new BigDecimal(ds.get().doubleValue());
+        BigDecimal bigDecimal = new BigDecimal(ds.get().toString());
```

Comment:
I think `BigDecimal.valueOf(1.0D)` should be used here.

http://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html#valueOf-double-